bowling/src/fileHandler.cpp

```
@@ -0,0 +1,45 @@
+#include "fileHandler.hpp"
+
+FileHandler::FileHandler(std::string fileName, FileHandler::FileAccess fileAccess)
+    : fileName_(fileName), fileAccess_(fileAccess)
+{
+    if (fileAccess_ == FileAccess::INPUT) {
+        fileStream_.open(fileName_, std::ios::in);
+    }
+    else {
+        fileStream_.open(fileName_, std::ios::out);
+    }
+}
+
+FileHandler::~FileHandler()
+{
+    if (isFileOpened()) {
+        fileStream_.close();
+    }
+}
+
+std::string FileHandler::readLine()
+{
+    if (!isFileOpened() || (fileAccess_ != FileAccess::INPUT)) {
+        return {};
```

good, you can throw an exception if you wish in this case